### DISCUSSION TITLE
Docker Swarm Deployment

### DISCUSSION THREAD

**Discussion by dadafan** [apache/pulsar#20038] [Q&A]
Is there a corresponding way to deploy Docker Swarm in the latest version 2.11.0 of Pulsar. I tried the old version of pulsar and successfully deployed it using Docker Swarm, but the latest version failed

**Comment by dadafan** [comment]
#6264 
This is a reference to previous versions that can be successful

**Comment by tisonkun** [comment]
> I tried the old version of pulsar and successfully deployed it using Docker Swarm, but the latest version failed

Can you create a reproduce repo for sharing what you exactly do and how it fails?
